components/cd-tag/cd-tag.css

```diff
@@ -0,0 +1,75 @@
+/**
+ * Base component styles for the collection of tags
+ */
+.cd-tags {
+  /* Override container class if you wish to provide specific styling to all tags */
+}
+
+/**
+ * Individual tag
+ *
+ * - margin is set in `rem` to ensure that multiple tag sizes visually line up
+ * - all other units are `em` so that setting `font-size` can scale them easily
+ */
+.cd-tag {
+  display: inline-block;
+  margin: 0.25rem;
+  padding: 0 .25em;
+  background-color: #5090cd; /* default is up for discussion */
+  color: #fff;
+  font-size: .625em;
```

Comment:
REMs please. 
I would bump this up to 0.75rem or 1rem until a designer says otherwise.

Reply:
This also destroys a "feature" of the component, which allows you to tag content of various font-sizes and have it automatically retain proportion. On GHO we had tags on page titles, but on HID it's on body-copy-sized text, etc.

Reply:
@rupl I meant to get back to this. I'm not suggesting we destroy the feature of the component at all. I didn't understand the details. This seems worth rewinding.